README.md

```diff
@@ -1,19 +1,25 @@
 <p align="center">
-<img width="300px" src="readme-style.png">
+  <img width="300px" src="readme-style.png">
+</p>
 
 <p align="center">
-<h3 align="center">Primer.style</h3>
+  <h3 align="center">Primer.style</h3>
+</p>
 
 <p align="center">
-<a href="https://styleguide.github.com/primer">Styleguide</a>
-    ·
-    <a href="https://github.com/primer">Primer org</a>
-    ·
-    <a href="https://spectrum.chat/primer">Community</a>
+  <a href="https://primer.style/css">Documentation</a>
```

Comment:
@emplums what do you think about the `README` links? Shall we a) change this _Documentation_ link also to point to _Team_ or b) remove it at all since there's already a _Team_ section below in https://github.com/primer/primer.style/commit/0ffb51b69809c2e0d40655987c1d3d733b216e4d#diff-04c6e90faac2675aa89e2176d2eec7d8R22?

Reply:
I think we can just remove it for now :)

Reply:
Sounds good! Removed in a031f86.